config/ftbquests/quests/chapters/bigger_reactors.snbt

```diff
@@ -0,0 +1,1084 @@
+				"- Absorption: The speed at which radiation is absorbed and converted to case heat"
+				"- Efficiency: How efficiently absorbed radiation is converted into heat."
+				"- Moderation: The effectiveness at which radiation is moderated."
+				"- Conductivity: How well heat is transferred from the fuel rods to the reactor casing."
+				""
+				"Typically, the rarer the material is, the better of a Moderator it is. In the ATM series, &dUnobtainium Blocks&r are considered one of the best moderators."
+				""
+				"Adding these inside of your reactor can increase the overall performance and output of your reactor. Try using diamond blocks in the early game!"
+			]
+			dependencies: ["4AD8363D7359A072"]
+			min_width: 300
+			id: "73362EDC984B8A0F"
+			tasks: [{
+				id: "3B3C22507E3FBACC"
+				type: "item"
+				item: "minecraft:diamond_block"
+				count: 3L
+			}]
+			rewards: [
+				{
+					id: "56D7ED254FC3A540"
+					type: "xp"
+					xp: 50
+				}
+				{
+					id: "6150F69F6D20EBFB"
+					type: "loot"
+					exclude_from_claim_all: true
+					table_id: 4196188979167302596L
+				}
+			]
+		}
+		{
+			title: "Converting our \"Passive\" Reactor"
+			x: 1.0d
+			y: 10.0d
+			subtitle: "To Make Steam!!"
+			description: [
+				"When a reactor is Passive, it just burns fuel directly into power. When you add a &dReactor Coolant Port&r to one of the reactor's walls, it converts the reactor to an Active reactor, which will instead burn fuel to heat fluids, like water into &9Steam&r. You cannot have a &cPower Tap&r and a Coolant Port on the same reactor. "
+				""
+				"Once placed, it will also convert the reactor's interface, showing more information about your reactor. You will now have a tank for the coolant, a tank for the exhaust, and can see the &9Exhaust Generation Rate&r. For example: If you're pumping in water, this will be the steam output."
+			]
+			dependencies: ["4415C9F8DA2D7E68"]
+			min_width: 300
+			id: "476755275B948A5F"
+			tasks: [{
+				id: "3A1B16362F885F1F"
+				type: "item"
+				item: "biggerreactors:reactor_coolant_port"
+				count: 2L
+			}]
+			rewards: [
+				{
+					id: "450F59D9CB0FEF7A"
+					type: "xp"
+					xp: 10
+				}
+				{
+					id: "44BF0BA518FE9E1D"
+					type: "random"
+					exclude_from_claim_all: true
+					table_id: 487623848494439020L
+				}
+			]
+		}
+		{
+			title: "Expanding Our Reactors"
+			x: 1.5d
+			y: 5.5d
+			description: [
+				"Reactors are multi-block structures, meaning you can create your own custom size!"
+				""
+				"The &lBiggest&r Reactor that you can make is 128x128x192. There are a lot of variables that contribute to the overall output of a reactor, so make sure to experiment!"
+				""
+				"Some overall tips:"
+				""
+				"The taller the reactor, the more fuel it can hold and burn because there are more fuel rods, meaning more overall power. This also means a higher &9burn rate&r."
+				""
+				"The wider the reactor, the more efficient it is. This means less consumption overall."
```

Comment:
Probably would be good to note that this is only true if you don't add more fuel rods when making it wider (while taller requires you to add more)

Reply:
Noted!